README.md

```diff
@@ -140,6 +140,9 @@ Complete all of these tasks on your local home machine.
   * Rackspace Cloud
 
             sudo pip install pyrax
+  * Microsoft Azure
+
+            sudo pip install msrest msrestazure azure==2.0.0rc5
```

Comment:
I did get some errors the first time though. I will clean and install again just to be sure.